src/pages/index.astro

```diff
@@ -1,50 +1,66 @@
 ---
-import HomeLayout from '../layouts/HomeLayout.astro';
+import BaseLayout from '../layouts/BaseLayout.astro';
```

Comment:
Can't call it HomeLayout anymore since the blog template also needs to use this.